src/main/java/com/zaxxer/hikari/metrics/prometheus/HikariCPCollector.java

```diff
@@ -63,14 +61,13 @@ void remove(String name)
       poolStatsMap.remove(name);
    }
 
-   private GaugeMetricFamily createGauge(String metric, String help,
-                                         Function<PoolStats, Integer> metricValueFunction)
+   private GaugeSnapshot createGauge(String metric, String help,
+                                                            Function<PoolStats, Integer> metricValueFunction)
    {
-      var metricFamily = new GaugeMetricFamily(metric, help, LABEL_NAMES);
-      poolStatsMap.forEach((k, v) -> metricFamily.addMetric(
-         Collections.singletonList(k),
-         metricValueFunction.apply(v)
+      Collection<GaugeSnapshot.GaugeDataPointSnapshot> gaugeDataPointSnapshots = new ArrayList<>();
+      poolStatsMap.forEach((k, v) -> gaugeDataPointSnapshots.add(
+         new GaugeSnapshot.GaugeDataPointSnapshot(metricValueFunction.apply(v), Labels.of(k), null)
```

Comment:
`Labels.of(k)` expects at least 2 arguments

Reply:
makes sense, I pushed the change suggestion